doc/doap-iframe.html

```diff
@@ -0,0 +1,19 @@
+<!--
+SPDX-FileCopyrightText: 2022 Melvin Keskin <melvo@olomono.de>
+
+SPDX-License-Identifier: CC0-1.0
+-->
+
+<html>
+	<head>
+		<script>
+			<!-- Dynamic resizing of the frame is needed because its size inside of the page is unknown before rendering it. -->
+			function resizeIframe(obj) {
+				obj.style.height = obj.contentWindow.document.documentElement.scrollHeight + 20 + 'px';
+			}
+		</script>
+	</head>
+	<body>
+		<iframe src="doap.xml" class="xeps" width="100%" height="100%" frameborder="0" onload="resizeIframe(this)"/>
+	</body>
+</html>
```

Comment:
why have you readded the `<html/>` tag? having `<html/>` inside of the main `<html/>` is not allowed.

Reply:
I can't find it on github anymore. I'm pretty sure I did push that (but it's of course possible that I somehow forgot it).